src/Umbraco.Web.UI/config/umbracoSettings.config

```diff
@@ -104,7 +104,8 @@
 
     <!-- Defines the default document type property used when adding properties in the back-office (if missing or empty, defaults to Textstring -->
     <defaultDocumentTypeProperty>Textstring</defaultDocumentTypeProperty>
-    
+
+    <showDeprecatedPropertyEditors>true</showDeprecatedPropertyEditors>
```

Comment:
OK my bad - just assumed where it was committed to the file that this was the value that we ship with.